src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -261,8 +261,8 @@ .maxstack  6
   IL_002f:  ldstr      ""y""
   IL_0034:  call       ""(int A, int B) Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<(int A, int B)>(string)""
   IL_0039:  ldloc.0
-  IL_003a:  stind.ref
-  IL_003b:  ret
+  IL_003a:  stobj      ""System.ValueTuple<int, int>""
```

Comment:
Thanks! We should either recognize missing ValueType to be a special type (that would make derived types structs) or report an error. Neither is happening. 

---
In reply to: [81822455](https://github.com/dotnet/roslyn/pull/14161#discussion_r81822455) [](ancestors = 81822455)